frontend/components/product-list/sections/FeaturedProductListSection.tsx

```diff
@@ -140,7 +130,8 @@ export function FeaturedProductListSection({
                   apiKey={ALGOLIA_API_KEY}
```

Comment:
@sctice-ifixit do you think it's a problem that we're exposing a key that has read access to the whole index? I don't believe we care _that_ much about the private data in Algolia, but filtering private data out client side seems like a no-go.

Reply:
Yeah, we should be generating a key server-side that has filters like `public=1` locked in. That will also make sure that we don't need to worry about specifying it as part of a product list page's filters.

Reply:
Hey @sctice-ifixit, just to be clear, are you referring to [this](https://www.algolia.com/doc/guides/building-search-ui/going-further/api-keys-security/react/) ? I've only found this key generated via API, is there also a way to generate the key using the Algolia dashboard? I've tried in the API keys section but it seems like I can't add filters restrictions when using the dashboard.

Reply:
We talked about this in person, but just for posterity, I'm referring to [secured API keys](https://www.algolia.com/doc/api-reference/api-methods/generate-secured-api-key/), which are generated on the backend using a search-only API key without any network communication.